doc/sphinx/proof-engine/tactics.rst

```diff
@@ -36,6 +36,18 @@ language will be described in Chapter :ref:`ltac`.
 Common elements of tactics
 --------------------------
 
+Reserved keywords
+~~~~~~~~~~~~~~~~~
+
+The tactics described in this chapter reserve the following keywords::
+
+  by using _eqn.
```

Comment:
I only see `IDENT "_eqn"` in the grammar, so it's not a token.

Reply:
```
Coq < Lemma _eqn: True.
1 subgoal

  ============================
  True

_eqn <
```

Reply:
Indeed. I copied it from the gallina chapter without verifying. Sorry.